src/parser/metrics/Complexity.ts

```diff
@@ -49,7 +50,16 @@ export class Complexity implements Metric {
 
     async calculate(parseFile: ParseFile, tree: Parser.Tree): Promise<MetricResult> {
         const queryBuilder = new QueryBuilder(parseFile, tree);
-        queryBuilder.setStatements(this.mccStatementsSuperSet);
+        if (parseFile.language === Languages.Java) {
+            //add query for instance init block in Java
+            queryBuilder.setStatements(
+                this.mccStatementsSuperSet.concat(
+                    new SimpleQueryStatement("(class_body (block)) @initBlock")
+                )
+            );
+        } else {
+            queryBuilder.setStatements(this.mccStatementsSuperSet);
+        }
```

Comment:
Isn't it possible like this to shorten the code?

```suggestion
        if (parseFile.language === Languages.Java) {
            //add query for instance init block in Java
            this.mccStatementsSuperSet.concat(
                new SimpleQueryStatement("(class_body (block)) @initBlock")
            );
        }
        queryBuilder.setStatements(this.mccStatementsSuperSet);
```

Reply:
No. `concat` actually returns a new array object that copies all elements. And that is on purpose, as mutating the class field (in this case `this.mccStatementsSuperSet`) like you seem to suggest here, e.g. by using `push` which mutates the existing array, would be a side effect that also affects further runs on other files. That was the exact issue that caused #65 and should be avoided therefore.